spring-cloud-dataflow-server-kubernetes-docs/pom.xml

```diff
@@ -58,14 +58,24 @@
 						<artifactId>asciidoctor-maven-plugin</artifactId>
 						<version>1.5.3</version>
 						<dependencies>
-						  <dependency>
-						    <!-- Needed to resolve URI includes -->
-						    <!-- See https://github.com/asciidoctor/asciidoctor.js/issues/153 -->
-						    <groupId>org.jruby</groupId>
-						    <artifactId>jruby-complete</artifactId>
-						    <version>9.0.0.0</version>
-						  </dependency>
+							<dependency>
+								<groupId>org.asciidoctor</groupId>
+								<artifactId>asciidoctorj-pdf</artifactId>
+								<version>1.5.0-alpha.11</version>
+							</dependency>
 						</dependencies>
+						<configuration>
+							<sourceDocumentName>index.adoc</sourceDocumentName>
+							<doctype>book</doctype>
+							<attributes>
+								<allow-uri-read>true</allow-uri-read>
+								<attribute-missing>warn</attribute-missing>
+								<project-version>${project.version}</project-version>
+								<project-artifactId>${project.artifactId}</project-artifactId>
+								<version-type-lowercase>${version-type-lowercase}</version-type-lowercase>
+								<snippets>${basedir}/target/generated-snippets</snippets>
```

Comment:
I think this one is not needed either, it is the destination directory of spring rest-docs generation, but that happens in SCDF core only (I understand you copy/pasted). I shall remove.